app/src/pages/inside/testPlansPage/testPlanDetailsPage/testPlanFolders/testPlanFolders.tsx

```
@@ -31,34 +29,48 @@ interface TestPlanFoldersProps {
+        testPlanRoute: `folder/${id}`,
+      },
+    });
+  }
   };
 
   return (
     <ExpandedOptions
-      activeFolderId={activeFolder}
+      activeFolderId={activeFolderId}
       folders={folders}
       instanceKey={TMS_INSTANCE_KEY.TEST_PLAN}
```

Biome (2.3.13)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)